src/app/shared/components/List/ListItem.tsx

```diff
@@ -141,7 +142,7 @@ export default function ListItem<T extends { id: string }>({
     }
 
     if (event.clientY + childHeight > innerHeight) {
-      y = y - childHeight;
+      y = y - childHeight + TOP_FIXED_HEIGHT;
```

Comment:
This solution breaks the behaviour of unfolding upwards when there is no space underneath.
I would look for something more along the lines of not overflowing the limits and if there is no space it shrinks but still scrolls.

Reply:
The solution applied is to fix the height at which the menu is displayed up or down so that it is not hidden.